src/openforms/contrib/zgw/clients/catalogi.py

```diff
@@ -2,7 +2,7 @@
 
 from zgw_consumers.nlx import NLXClient
 
-from .utils import pagination_helper
+from openforms.utils.api_clients import pagination_helper
```

Comment:
FYI, this will be included in the next version of zgw-consumers!

Reply:
Ah nice to hear, I've had to use some `typing_extensions` features in our current implementation (not in draft this time :stuck_out_tongue:) but this is not a big deal without it